spec.md

```diff
@@ -223,41 +228,55 @@ License: GPL-3.0-or-later
 
 ## Order of precedence
 
-Copyright and Licensing Information is considered according to the
-following order of precedence:
+If a Commentable File contains Copyright and Licensing Information but also has
+an adjacent `.license` file, then the Copyright and Licensing Information
+defined in the `.license` file takes precedence, and the Commentable File's
+contents are ignored.
 
-1. Information defined in the `.license` file.
-2. Information defined in the Commentable File.
-3. Information defined in `.reuse/dep5`.
+Copyright and Licensing Information defined in `.reuse/dep5` is aggregated with
+the Copyright and Licensing Information found in the covered files or in their
+adjacent `.license` files. For clarity, this means that if the SPDX License
+Expressions in a file's comment header and in the section for that file in
+`.reuse/dep5` do not agree with each other, then both SPDX License Expressions
```

Comment:
I find this a bit weird, TBH. Example:

- source code file says: `MIT`
- its `.license` file says: `CC0`
- `.reuse/dep5` (in the future `reuse.yaml`, I suppose) says: `* GPL-2.0-only`

Ergo `reuse` says: `CC0 AND GPL-2.0-only`

But in my example dep5 used a very broad-scoped glob, so I would not really trust it. And especially not over what the source code header itself says.

I know we had a very very long thread about this and I’m not trying to re-start the discussion. It is just that I don’t remember the outcome of that thread being such.

Reply:
You're correct. That thread is [here](https://github.com/fsfe/reuse-tool/issues/779). Especially note the use case in my first post.

The situation is Not Good. But I think it's valuable to do a relatively minor 3.2 release of the spec that finally defines the (formally) unspecified behaviour of the tool.

Then later we can do a 4.0 release that completely fixes the order of precedence.

Do you think this section should include a notice à la 'by the way, this is going to change in the future'? I considered it, but decided against it with no good rationale.

Reply:
I wonder if it makes sense to include it here then, if it’s going to change / break anyway in 4.0.

Reply:
Because we're keeping DEP5 compatibility in the meantime, I'm keeping this as-is.